libs/frontend/shared/generic-page/src/lib/generic-page/generic-page.component.html

```diff
@@ -2,8 +2,10 @@
   <div *ngIf="!isLoading">
     <div class="top">
       <div class="left title">{{ title }}</div>
-      <div class="right">
-        <button class="button" (click)="onButtonClick()">{{ buttonText }}</button>
+      <div *ngIf="withButton">
+        <div class="right">
+          <button class="button" (click)="onButtonClick()">{{ buttonText }}</button>
+        </div>
       </div>
```

Comment:
Pourquoi avoir ajoute une div inutile ?
```suggestion
      <div class="right">
        <button class="button" (click)="onButtonClick()">{{ buttonText }}</button>
      </div>
```

Reply:
Bro ? le ngIf bro